pages/TheLearnerView.html

```diff
@@ -51,7 +51,7 @@ <h3>
     dashboards of the exploration editors.
   </p>
 
-  <img src="static/images/explorationFeedback.png" width="500">
+  <img src="static/images/explorationFeedback.png" width="500" alt="An exploration card with a white background, along with a 'message' icon on the top-right corner. On-clicking this icon, a text-area is prompted through which one can submit feedback.">
```

Comment:
prompted --> displayed

text-area --> text area. Please remove superfluous hyphens in other places.